.github/workflows/pythonpackage.yml

```diff
@@ -31,7 +31,34 @@ jobs:
     - name: Python License Check
       run: |
         python -m detection_rules dev license-check
-        
+
+    - name: Unit tests
+      run: |
+        python -m detection_rules test
+
+    - name: Get Backport Status
+      id: get_backport_status
+      uses: fjogeleit/http-request-action@v1
+      with:
+        url: "https://api.github.com/repos/elastic/detection-rules/actions/workflows/backport.yml/runs?per_page=1"
+        method: 'GET'
+
+    - name: Check Backport Status
+      uses: actions/github-script@v6
+      with:
+        script: |
+          const workflow_status = ${{ toJSON(fromJSON(steps.get_backport_status.outputs.response).workflow_runs[0].status) }}
```

Comment:
I don't think this approach will work as intended. This will always get the latest status of the last run `backport` flow, but that runs on every label change in every PR, with the actual backporting only occurring on merges to main.

I think it might be more dependable to consult the results of the `checks` for the respective branches directly. GH actions lets you share info across workflows, so we could consult the matrix of branches that way, or just list them in an etc txt file and have the different actions consult that

Reply:
Right. I think the same approach will work only we change the url to point to the right branch url and check workflow. Ill also look into sharing info across workflows. Seems neat!